.changeset/gentle-deers-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Allow passing down withSearch prop to EntityTechdocsContent component
```

Comment:
```suggestion
Allow passing down `withSearch` prop to `EntityTechdocsContent` component
```

The above should fix the Vale errors. Can we also maybe add some details as to what this does for the end user?